explainer.md

```diff
@@ -850,11 +850,13 @@ Using existing manifest_url/start_url would be bad because there are multiple us
 
 With the primary goal of ensuring having a stable identifier to support various use cases of updating app's metadata. Using **start_url\_origin + id** is preferred as this is the most stable option.
 
-For the default field to use, **start_url** is preferred.
+For the default field to use, **start_url** is preferred. A fallback default id is supported to allow smooth transition from existing platforms where an id field doesn't exist in the manifest. Therefore when considering which default field to choose, being backward compatible is the key consideration.
```

Comment:
Maybe move the first part (For the default field to use, **start_url** is preferred.) below.